### DISCUSSION TITLE
Ignoring Repository Renovate Configuration

### DISCUSSION THREAD

**Discussion by a1flecke** [renovatebot/renovate#33011] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab, latest

### Please tell us more about your question or problem

We are looking to redo our renovate rollout.  It has been humming along for a few years and we now have thousands of projects using it.  The effectiveness of renovate has been hampered by both initial decisions on our part and on local configuration decisions that the project teams made that were wrong.

We would like to "redo renovate" and move to a more centralized configuration model.  Our hope is to not have to touch thousands of repositories to pull this off.  Is there a way to direct renovate to ignore the repository configuration file?

Example of the file that we want to ignore.  Globally we have decided that minors in common frameworks like Spring Boot or Rails should be treated as majors.   In this case the local config too broadly allow auto-merging of minor and patches by making the following change which trumps the global decision.

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "local>our-org/renovate-config"
  ],
  "packageRules": [
    {
      "matchUpdateTypes": ["minor", "patch"],
      "automerge": true
    }
  ]
}

```

Is there a way to run Renovate to ignore the repository configuration or to "start over" with a new configuration file? 

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
If you set `requireConfig=ignored` then repo config will be completely ignored

**Reply by a1flecke** [reply]
Thanks!  This is what I am looking for.
